src/commands/image/imageSource/buildImageInAzure/ImageNameStep.ts

```diff
@@ -4,16 +4,21 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { AzureWizardPromptStep } from "@microsoft/vscode-azext-utils";
-import { URI, Utils } from "vscode-uri";
 import { localize } from "../../../../utils/localize";
 import { validateUtils } from "../../../../utils/validateUtils";
-import { BuildImageInAzureImageSourceContext } from "./BuildImageInAzureContext";
+import { CreateContainerAppContext } from "../../../createContainerApp/CreateContainerAppContext";
+import type { BuildImageInAzureImageSourceContext } from "./BuildImageInAzureImageSourceContext";
 
 const maxImageNameLength: number = 46;
 
 export class ImageNameStep extends AzureWizardPromptStep<BuildImageInAzureImageSourceContext> {
     public async prompt(context: BuildImageInAzureImageSourceContext): Promise<void> {
-        const suggestedImageName = await getSuggestedName(context, context.rootFolder.name);
+        const suggestedImageName = ImageNameStep.getTimestampedImageName(
+            context.containerApp?.name ||
+            // Step is also technically reachable from the `createContainerApp` entry point
+            (context as CreateContainerAppContext).newContainerAppName ||
+            'hello-world'
```

Comment:
Mark one of these as `nonNullValue` lol. I don't want this random 'hello-world' in here.